pkg/transparentproxy/config/config.go

```diff
@@ -57,6 +72,27 @@ func (c *Owner) Set(s string) error {
 	)
 }
 
+func (c *Owner) UnmarshalJSON(bs []byte) error {
+	var jsonValue interface{}
+
+	if err := json.Unmarshal(bs, &jsonValue); err != nil {
+		return err
+	}
+
+	switch typedValue := jsonValue.(type) {
+	case string, float64:
+		return c.Set(fmt.Sprint(jsonValue))
+	case map[string]interface{}:
```

Comment:
This looks like a magic. Why do we need to handle map and key "uid" specially? 
May be a comment is helpful if it's not a big deal.

Reply:
That was to allow user to provide a value with an object with a `uid` key as basically it's how the underlying structure looks like, but I got rid of it